img/clicked.svg

```diff
@@ -0,0 +1,86 @@
+		C512,236.584,486.832,184.408,449.056,154.528z"/>
+</g>
+<path style="fill:#F79220;" d="M120,296c0-79,44.432-145.264,104-163.112c-10.28-3.072-20.96-4.888-32-4.888
+	c-74.992,0-136,75.36-136,168s61.008,168,136,168c11.04,0,21.72-1.816,32-4.888C164.432,441.264,120,375,120,296z"/>
+<path style="fill:#FBAC43;" d="M256,128c-74.992,0-136,75.36-136,168s61.008,168,136,168s136-75.36,136-168S330.992,128,256,128z"/>
+<path style="fill:#F79220;" d="M312,128c-9.6,0-18.96,1.272-28,3.616c61.6,16,108,83.6,108,164.384s-46.4,148.392-108,164.384
+	c9.04,2.352,18.4,3.616,28,3.616c74.992,0,136-75.36,136-168S386.992,128,312,128z"/>
+<g>
+	<path style="fill:#E48225;" d="M385.056,154.528C409.112,184.84,424,226.296,424,272c0,69.08-33.944,128.512-82.304,154.296
+		c-16.496,16.584-36.168,28.496-57.728,34.096C293.016,462.752,302.392,464,312,464c74.992,0,136-75.36,136-168
+		C448,236.584,422.832,184.408,385.056,154.528z"/>
+	<path style="fill:#E48225;" d="M184.496,438.64C179.08,439.456,173.592,440,168,440c-26.896,0-51.92-9.824-73.056-26.528
+		C119.648,444.608,154.008,464,192,464c11.04,0,21.72-1.816,32-4.888C209.832,454.856,196.552,447.888,184.496,438.64z"/>
+</g>
+<path style="fill:#6E421B;" d="M296,304v16c0,8.832-7.168,16-16,16s-16-7.168-16-16v-16h-48c-17.672,0-32,14.328-32,32
+	s14.328,32,32,32v-16c0-8.832,7.168-16,16-16s16,7.168,16,16v16h48c17.672,0,32-14.328,32-32S313.672,304,296,304z"/>
+<g>
+	<path style="fill:#59340F;" d="M327.352,342.448C327.776,340.36,328,338.2,328,336c0-17.672-14.328-32-32-32v12.888
+		C311.464,316.888,324.368,327.856,327.352,342.448z"/>
+	<path style="fill:#59340F;" d="M264,332.888c0,8.832,7.168,16,16,16s16-7.168,16-16V320c0,8.832-7.168,16-16,16s-16-7.168-16-16
+		V332.888z"/>
+	<path style="fill:#59340F;" d="M216,316.888h48V304h-48c-17.672,0-32,14.328-32,32c0,2.2,0.224,4.36,0.648,6.448
+		C187.632,327.856,200.536,316.888,216,316.888z"/>
+</g>
+<g>
+	<circle style="fill:#6E421B;" cx="216" cy="248" r="24"/>
+	<circle style="fill:#6E421B;" cx="296" cy="248" r="24"/>
+</g>
+<g>
+	<path style="fill:#59340F;" d="M296,232c11.88,0,21.68,8.664,23.592,20c0.232-1.312,0.408-2.632,0.408-4c0-13.248-10.752-24-24-24
+		c-13.256,0-24,10.752-24,24c0,1.368,0.176,2.688,0.408,4C274.312,240.664,284.112,232,296,232z"/>
+	<path style="fill:#59340F;" d="M216,232c11.88,0,21.68,8.664,23.592,20c0.232-1.312,0.408-2.632,0.408-4c0-13.248-10.752-24-24-24
+		c-13.256,0-24,10.752-24,24c0,1.368,0.176,2.688,0.408,4C194.312,240.664,204.112,232,216,232z"/>
+</g>
+<path style="fill:#F79D32;" d="M329.056,154.528C353.112,184.84,368,226.296,368,272c0,92.64-61.008,168-136,168
+	c-26.896,0-51.92-9.824-73.056-26.528C183.648,444.608,218.008,464,256,464c74.992,0,136-75.36,136-168
+	C392,236.584,366.832,184.408,329.056,154.528z"/>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>

```

Comment:
This probably isn't your SVG code, but these hanging `g`'s just really grind my gears haha. no big deal